lib/enum/arrays.rb

```diff
@@ -21,4 +26,8 @@ def lowercase_string_elements_as_uppercase
   def capitalized_elements
   end
 
+  def array
```

Comment:
Not sure why this is here - you already have `@array` from the initialized method.